### ISSUE TITLE
Define "Member/member"

### ISSUE THREAD

**Issue by sparrell** [oasis-board-process-committee/TC-Process#6]
As a future work item, the Process Committee may want to consider a process document on “member” (meaning an individual who participates in the work of OASIS) vs “Member” (meaning the legal entities defined as Members per the OASIS bylaws which for the most part are companies and government agencies).  We use them as synonyms too often and they are very different depending on the context. Note you may not want to use my little “m”, big “M” as the two words to distinguish. But we too frequently in my opinion use them ambiguously when we shouldn’t. Without opening a can of worms (at least don’t open until after we approve the current draft), one example would be the use of the word in the vulnerability policy and process documents.

I agree it’s a picky issue, but important nonetheless since we are a 501c6 corporation. 501c3 nonprofits operate for the ‘public good’. 501c6 nonprofits (ie OASIS) operate for the benefit of the “Members” – the legal entities as defined by the bylaws (ie the companies/agencies that get 1 vote apiece to elect us). So I think it’s important to distinguish between individuals doing volunteer work in a repo from the corporations that are the Membership of OASIS, particularly in sentences like “have an obligation” wrt topics that might involve regulation or litigation.

**Comment by jordan2175** [comment]
I agree. This is a an issue and can typically cause a lot of problems. The reality is, organizations often pay for the membership but let their employees do the work they want to do. There are times when employees have to toe the company line, but that is not always the case. I have seen this both ways. But making it clear with an individual can do versus what the paying organization can do, would be helpful.

**Comment by sparrell** [comment]
I think it appropriate to spend some effort on this issue given a "Member" meeting coming up. Having said that, I'm not sure what next step is.

**Comment by jordan2175** [comment]
All of these issues will be discussed in one of the upcoming Process Committee meetings. The group will then make a decision on what to do, if anything.
